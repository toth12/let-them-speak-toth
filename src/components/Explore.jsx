import React from 'react';
import Hero from './Hero';
import { Link } from 'react-router-dom';

const Explore = props => (
  <div>
    <Hero text='Explore' />
    <div className='container'>
      <h2>Subheading here for page section</h2>
      <p>Lorem ipsum dolor sit amet, arcu vulputate egestas dolor wisi, fugiat vestibulum etiam mattis sit, curabitur elit nulla vel, mus quis elit porttitor nulla. Amet elit in convallis, metus nam at sed, ipsum phasellus a dapibus, ornare in massa pharetra. Montes elementum pede integer ac, pulvinar aliquam non in augue, orci nulla eros nulla. Vulputate ut sed eget eu, dolor aliquam velit quisque a. Fermentum venenatis rhoncus vitae congue, suspendisse potenti pellentesque vestibulum.</p>
      <p>Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse. A sit, vehicula eleifend, per massa vitae. Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse.</p>
      <div className='button-trio'>
        <Link className='button' to='/search'>Keyword Search</Link>
        <Link className='button' to='/contents'>Table of Contents</Link>
      </div>
    </div>
  </div>
)

export default Explore;