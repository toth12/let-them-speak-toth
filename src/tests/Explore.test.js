import React from 'react';
import Explore from '../components/Explore';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Explore/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});