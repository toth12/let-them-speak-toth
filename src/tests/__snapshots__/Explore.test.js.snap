// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    className="hero background-image"
  >
    <h1>
      Testimonies
    </h1>
  </div>
  <div
    className="container"
  >
    <p>
      In addition to search, a traditional table of contents supports the exploration of interviews. This leads readers to complete transcripts, and to the listening or watching some but not all interviews. The table of contents is also filterable in terms of certain metadata.
    </p>
    <p>
      Transcripts are not fully accurate; they might feature occasional errors, omissions, odd formatting and deviations from the original audio / video tapes. The goal of a data edition is not to produce a completely accurate publication of data. Due to the volume of testimonies data and the fact that interviews were recorded and transcribed by many different projects using different and not always consistent transcription standards, this project cannot offer complete accuracy. Interview transcripts are undergoing a basic quality control that tries to find erroneous transcripts.  In addition to transcript errors, historical inaccuracies may be present as these are personal recollections and reflections of individual survivors and witnesses. Please take this into consideration when reading transcripts and contact the editor with suggestions for improvement. You can read more about data processing and quality control in the 
      <a
        href="/methods"
      >
        Methodology
      </a>
       section.
      <br />
      <br />
    </p>
    <div
      className="center-text"
    >
      <a
        className="button brown"
        href="/contents"
        onClick={[Function]}
      >
        Let's explore
      </a>
    </div>
  </div>
</div>
`;
